lib/api/controllers/authController.js

```diff
@@ -98,7 +98,9 @@ class AuthController extends BaseController {
   logout(request) {
     assertIsAuthenticated(this.anonymousId, request);
 
-    return this.kuzzle.repositories.token.expire(request.context.token)
+    return request.input.args.global
```

Comment:
You should use [BaseController.tryGetBoolean](https://github.com/kuzzleio/kuzzle/blob/master/lib/api/controllers/baseController.js#L71)